generated/nidaqmx/nidaqmx_service.h

```diff
@@ -23,14 +23,23 @@
 
 namespace nidaqmx_grpc {
 
+struct NiDAQmxFeatureToggles
```

Comment:
Note: this code is moved around slightly so that the toggles can be created (and checked) in the registrar methods before creating the service.